README.md

```diff
@@ -65,7 +47,94 @@ $loop->addPeriodicTimer(5, function () {
 $loop->run();
 ```
 
-**Note:** The factory is just for convenience. It tries to pick the best
-available implementation. Libraries `SHOULD` allow the user to inject an
-instance of the loop. They `MAY` use the factory when the user did not supply
-a loop.
+## Usage
+
+In order for async based libraries to be interoperable, they need to use the
+same event loop. This component provides a common `LoopInterface` that any
+library can target. This allows them to be used in the same loop, with one
+single `run()` call that is controlled by the user.
```

Comment:
I've moved the paragraph back to the top and replaced it here with a short introduction.